test/unit/merge.js

```diff
@@ -25,7 +25,8 @@ describe('Merge', function () {
   beforeEach('instanciate merge', function () {
     this.side = 'local'
     this.merge = new Merge(this.pouch)
-    this.merge[this.side] = {renameConflictingDocAsync: sinon.stub().resolves()}
+    this.merge.local = {renameConflictingDocAsync: sinon.stub().resolves()}
+    this.merge.remote = {renameConflictingDocAsync: sinon.stub().resolves()}
```

Comment:
We had to stub Remote in order for our unit test to show some useful failure message.

Reply:
(could benefit other unit tests too)